### PR TITLE
fix(veracode):[TRI-397] Add Veracode integration

### PR DESCRIPTION
**Pull request by mkanal**

### CODE REVIEW

File: .github/workflows/VeraCode.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'corretto'
```
Comment (ds-jhartmann) [inline .github/workflows/VeraCode.yml:18]: In the workflow CI-main we use distribution 'adopt'. Might want to use this here too.

File: .github/workflows/VeraCode.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+      - name: Set up Maven
```
Comment (ds-jhartmann) [inline .github/workflows/VeraCode.yml:20]: Re-use the action from our CI-main workflow 'Cache maven packages'

**Review by ds-jhartmann — CHANGES_REQUESTED** [review]
I requested two changes, the rest looks good to me.

File: .github/workflows/VeraCode.yml (hunk 1 above)
Comment (ds-agenz) [reply .github/workflows/VeraCode.yml]: thanks., i changed it

File: .github/workflows/VeraCode.yml (hunk 2 above)
Comment (ds-agenz) [reply .github/workflows/VeraCode.yml:20]: I added the caching

**Review by ds-jhartmann — APPROVED** [review]

**Pull request merged**